nntrainer/src/conv2d_layer.cpp

```diff
@@ -229,9 +229,9 @@ sharedConstTensor Conv2DLayer::backwarding(sharedConstTensor derivative,
       input.getAddress(b * input.getDim().getFeatureLen()));
 
     status = conv2d_gemm(
-      derivative->getAddress(b * derivative->getDim().getFeatureLen()),
-      TensorDim(1, derivative->channel(), derivative->height(),
-                derivative->width()),
+      derivative[0]->getAddress(b * derivative[0]->getDim().getFeatureLen()),
```

Comment:
I think it might be more readable to get `derivative[0]` out separately.

Reply:
done.